x-pack/plugin/security/src/main/java/org/elasticsearch/xpack/security/authc/ApiKeyService.java

```diff
@@ -375,18 +377,14 @@ public void getRoleForApiKey(Authentication authentication, ActionListener<ApiKe
         }
     }
 
-    public BytesReference getRoleDescriptorsBytesForApiKey(Authentication authentication, boolean limitedBy) {
+    public Tuple<String, BytesReference> getApiKeyIdAndBytes(Authentication authentication, boolean limitedBy) {
```

Comment:
```suggestion
    public Tuple<String, BytesReference> getApiKeyIdAndRoleBytes(Authentication authentication, boolean limitedBy) {
```
(or role descriptor, or ...)
But `Bytes` to me implies the whole API key JSON or something like that.